### PR TITLE
Update version and CHANGELOG for new release: 0.9.0.

### PR DESCRIPTION
**Pull request by tdenewiler**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,12 +10,48 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
...
 
+### Fixed
+
+### Removed
+
+## v0.9.0 - 2022-09-12
+
+### Added
+
+- The new `--level` flag can be set on the command line and will override all other levels, even non-default levels
```
Comment (xydesa) [inline CHANGELOG.md:21]: we should add this (and the timing flag) to the README as well.  I forgot to do that as part of the ticket.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,15 +10,55 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
...
+  specified in a `--profile` flag when running Statick.
+  The expectation is that a user setting the `--level` flag will explicitly want that level for the entire Statick run
+  (single package or multiple packages in a workspace).
+  If separate levels are desired per package then the user should not use the `--level` flag. (#429, #436)
+- Ubuntu 22.04 is now included in the main test environment matrix when running GitHub Actions. (#444)
+- The `--timing` flag will print timing information to the console after a Statick run.
```
Comment (gregtkogut) [inline CHANGELOG.md:27]: Is it 'timings' and not 'timing?'

Per usage string:

` [--timings]`

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,15 +10,55 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
...
+
 - Default behavior for Statick will now run all available discovery plugins, and run all tool plugins where
-  their desired source files are available, then output results only on the terminal. (#432, #435)
+  their desired source files are available, then output results only on the terminal.
   The old default behavior was to run the "sei_cert" profile, this is still doable via either of the
-  following arguments: `--profile sei_cert.yaml` or `--level sei_cert`
-
-### Fixed
+  following arguments: `--profile sei_cert.yaml` or `--level sei_cert`. (#432, #435)
+- When running unit tests with tox Statick uses pytest-flake8.
```
Comment (gregtkogut) [inline CHANGELOG.md:36]:
```suggestion
- When running unit tests with tox, Statick uses pytest-flake8.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,15 +10,55 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
...
+  A recent bug upstream causes issues when using the latest version of pytest-flake8.
```
Comment (gregtkogut) [inline CHANGELOG.md:37]:
```suggestion
  A recent upstream bug causes issues when using the latest version of pytest-flake8.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -10,15 +10,55 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
...
+  Required removing a mypy ignore comment. (#437)
+- The ROS discovery plugin was setting the file type of the package to a boolean value rather than a string describing
+  the actual file type.
+  Mixing types between packages caused bugs in tool plugins.
+  The ROS discovery plugin now acts consistently with other discovery plugins. (#439)
 
 ### Removed
 
+- Usage of `inherits_from` flag in configuration files as a string is no longer supported.
+  The levels specified in `inherits_from` must be in list format now.
```
Comment (gregtkogut) [inline CHANGELOG.md:59]:
```suggestion
  The levels specified in `inherits_from` must now be in list format.
```

File: CHANGELOG.md (hunk 2 above)
Comment (gregtkogut) [inline CHANGELOG.md:27]:
```suggestion
- The `--timings` flag will print timing information to the console after a Statick run.
```

**Review by gregtkogut — CHANGES_REQUESTED** [review]
LGTM

**Review by gregtkogut — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (tdenewiler) [reply CHANGELOG.md:21]: [419aa61](https://github.com/sscpac/statick/pull/446/commits/419aa61bd2ec56ed4f98effcc6043f06130a0332)

File: README.md [hunk 6]
```diff
@@ -271,6 +274,10 @@ Carnegie Mellon University Software Engineering Institute
 The rules and flags can be found in the
 [SEI CERT C/C++ Analyzers](https://wiki.sei.cmu.edu/confluence/display/cplusplus/CC.+Analyzers) chapter.
 
+Using the `--level` flag when running Statick will result in that specific level running for all packages regardless
+of what the `--profile` is set to.
+The speicified _level_ from the `--level` flag must exist in the default configuration file or a custom configuration file.
```
Comment (xydesa) [inline README.md:279]:
```suggestion
The specified _level_ from the `--level` flag must exist in the default configuration file or a custom configuration file.
```

**Review by xydesa — APPROVED** [review]

**Pull request merged**